plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend

```diff
@@ -366,6 +371,35 @@ class TextualScenarioValidator extends AbstractTextualScenarioValidator {
 		}
 	}
 	
+	@Check
+	def checkLostMessage(LostMessage message) {
+		// check arm timer could be used in opened diagram

```

Comment:
Is it "lost message" instead of "arm timer"?

Reply:
yes